src/components/SwapTokenInput.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import { TextField } from "@mui/material"
+
+export default function SwapTokenInput(): React.ReactElement {
+  return (
+    <div>
+      <TextField name="swapTokenInput" />
+    </div>
+  )
```

Comment:
This file seems unnecessary, is there anything wrong with just using `TextField` directly in the parent component?

Reply:
Oh, Right.
This file is not necessary in this PR.